app/Http/Controllers/mainController.php

```diff
@@ -8,125 +8,151 @@
             }
         }
     }
-
-    function logout()
-    {
+    function logout(){
         //Just killing the session here :D
-        if (session()->has('LoggedUser')) {
+        if(session()->has('LoggedUser')) {
             session()->pull('LoggedUser');
             return redirect('login');
         }
     }
-
-
     //WARNING!!
     //Remember all functions added here are the ones inside web authCheck!
-
-    function profile()
-    {
-        $data = ['LoggedUserInfo' => User::where('id', '=', session('LoggedUser'))->first()];
-
+    function profile() {
+        $data = ['LoggedUserInfo'=>User::where('id','=', session('LoggedUser'))->first()];
         return view('user.profile', $data);
     }
-
-    function Settings()
-    {
-        $data = ['LoggedUserInfo' => User::where('id', '=', session('LoggedUser'))->first()];
-
+    function Settings() {
+        $data = ['LoggedUserInfo'=>User::where('id','=', session('LoggedUser'))->first()];
         return view('user.settings', $data);
     }
     //Thommi - dsjfhdsjfhdsfjs
-    function myWatchs()
-    {
-        $data = ['LoggedUserInfo' => User::where('id', '=', session('LoggedUser'))->first()];
-
+    function myWatchs() {
+        $data = ['LoggedUserInfo'=>User::where('id','=', session('LoggedUser'))->first()];
         return view('user.mywatchs', $data);
     }
-
-    function myRatings()
-    {
-        $data = ['LoggedUserInfo' => User::where('id', '=', session('LoggedUser'))->first()];
-
+    function myRatings() {
+        $data = ['LoggedUserInfo'=>User::where('id','=', session('LoggedUser'))->first()];
         return view('user.myratings', $data);
     }
+    function myMovies() {
+        $data = ['LoggedUserInfo'=>User::where('id','=', session('LoggedUser'))->first()];
+        return view('user.mymovies', $data);
+    }
+    //WARNING!!
+    //Remember all functions added here are the ones inside web authCheck!
+    //SELECT * FROM genre_movie INNER JOIN movies ON genre_movie.genre_id = movies.id ORDER BY genre_movie.genre_id DESC
+    public function getInfo($id) {
+        $page = DB::select("SELECT  movies.id,
+                                    movies.title,
+                                    movies.director,
+                                    movies.description,
+                                    movies.writer,
+                                    movies.year,
+                                    movies.img,
+                                    group_concat(concat(' <a href=''/genre/' , genres.genre , ''' class=''hover:underline italic''>' , genres.genre , '</a>')) AS genre
+                                    FROM movies
+                                    INNER JOIN genre_movie ON genre_movie.movie_id = movies.id
+                                    INNER JOIN genres ON genre_movie.genre_id = genres.id WHERE movies.id = $id
+                                    GROUP BY movies.id, movies.title, movies.year");
+        // dd($page);
```

Comment:
delete the dump die

Reply:
fixar de DD, fel av mig av att glömma radera dem.